src/selectors/host/selectors.js

```diff
@@ -1,3 +1,16 @@
 import { get } from 'lodash';
 
-export const getHostStatus = host => get(host, ['metadata', 'labels', 'metalkube.org/operational-status']);
+import { HOST_STATUS_TO_TEXT } from '../../utils/status/host/constants';
+
+export const getHostStatus = host => {
+  // Returns a status string based on the available host information.
+  const operationalStatus = get(host, 'status.operationalStatus');
+  const provisioningState = get(host, 'status.provisioning.state');
+
+  return [provisioningState || operationalStatus];
```

Comment:
I think the brackets are not needed here.

Reply:
Done, thanks